app/views/components/_filter_section.html.erb

```diff
@@ -4,7 +4,7 @@
   shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
   open ||= nil
   status_text ||= ""
-  heading_text ||= ""
+  visually_hidden_heading_prefix ||= ""
```

Comment:
Note: We already check on line 3 whether `heading_text` is given, so we don't need to assign it anything here.